THIRD-PARTY-PROGRAMS-IPP

```diff
@@ -0,0 +1,456 @@
+Intel® Integrated Performance Primitives (Intel® IPP) Third Party Programs File
```

Comment:
what is point of using capital letters in filenames? 
Do we have a guidelines here?

Reply:
related question: why these files do not have a .txt extension? Is this because of the new guidelines?